source/image-handler/test/image-handler/crop.spec.ts

```diff
@@ -10,16 +10,21 @@ import { ImageEdits, StatusCodes } from "../../lib";
 const s3Client = new S3();
```

Comment:
For the crop tests in crop.spec.ts, mocks are not needed because those tests are not using the S3 or Rekognition services.

*s3Client = new S3()* and *rekognitionClient = new Rekognition()* are creating instances of the service clients but, in these cases, this does not attempt to reach the service endpoints.

That was confirmed by testing with network access turned off and with an explicit invalid region being set and those tests continuing to run successfully.

We do have mocks supported in mock.ts for when service behaviors have to be emulated in other tests.